crates/db_schema/replaceable_schema/triggers.sql

```diff
@@ -41,18 +28,18 @@ BEGIN
         CALL r.create_triggers ('thing_actions', $$
             BEGIN
                 WITH thing_diff AS ( UPDATE
-                        thing_aggregates AS a
+                        thing AS a
                     SET
                         score = a.score + diff.upvotes - diff.downvotes, upvotes = a.upvotes + diff.upvotes, downvotes = a.downvotes + diff.downvotes, controversy_rank = r.controversy_rank ((a.upvotes + diff.upvotes)::numeric, (a.downvotes + diff.downvotes)::numeric)
                     FROM (
                         SELECT
                             (thing_actions).thing_id, coalesce(sum(count_diff) FILTER (WHERE (thing_actions).like_score = 1), 0) AS upvotes, coalesce(sum(count_diff) FILTER (WHERE (thing_actions).like_score != 1), 0) AS downvotes FROM select_old_and_new_rows AS old_and_new_rows
                 WHERE (thing_actions).like_score IS NOT NULL GROUP BY (thing_actions).thing_id) AS diff
             WHERE
-                a.thing_id = diff.thing_id
+                a.id = diff.thing_id
                     AND (diff.upvotes, diff.downvotes) != (0, 0)
                 RETURNING
-                    r.creator_id_from_thing_aggregates (a.*) AS creator_id, diff.upvotes - diff.downvotes AS score)
+                    a.creator_id AS creator_id, diff.upvotes - diff.downvotes AS score)
```

Comment:
I was quite confused by this trigger because it actually replaces `thing` in the sql code with `post`/`comment`, and so kept referring to `post_aggregates` when that was already removed. I fixed it and ran into another problem: the trigger for comment updates the comment table, so it fires another trigger, and ultimately throws "stack depth limit exceeded". I fixed this by adding `WHEN (pg_trigger_depth( ) = 0 )` to create_triggers()